keps/sig-network/0752-endpointslices/kep.yaml

```diff
@@ -15,10 +16,10 @@ approvers:
 prr-approvers:
   - "@wojtek-t"
```

Comment:
requires PRR approval for GA: see e.g.:
https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-network/2200.yaml

Please create this file and assign to PRR approver after having SIG network leads buy-in.

Reply:
Created the new file. Thanks for pointing that out!